### ISSUE TITLE
Wrong background yellow on Windows 10 1903

### ISSUE THREAD

**Issue by kotori2** [shiena/ansicolor#14]
Not sure whether it is a Windows issue or ansicolor issue.
Light yellow and white works fine, but yellow broke.
![image](https://user-images.githubusercontent.com/17663689/62871101-64a24b80-bd4d-11e9-911a-03cd9ede3fc1.png)
```
package main

import (
	"io"
	"os"
	"sync"

	"github.com/shiena/ansicolor"
)

type logWriter struct {
	sync.Mutex
	writer io.Writer
}

func newLogWriter(wr io.Writer) *logWriter {
	return &logWriter{writer: wr}
}

func main() {
	w := newLogWriter(ansicolor.NewAnsiColorWriter(os.Stdout))
	w.writer.Write([]byte("\x1b[43m Yellow"))
	w.writer.Write([]byte("\x1b[103m Light Yellow"))
	w.writer.Write([]byte("\x1b[47m White \x1b[0m"))
}
```

**Comment by shiena** [comment]
@kotori2 
It's probably a console color issue.
https://devblogs.microsoft.com/commandline/updating-the-windows-console-colors/

There is a tool for changing the color palette.
However, in my environment, it worked with cmd and PowerShell6, but the background color of PowerShell5 became strange.
https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool
https://github.com/microsoft/terminal/releases/tag/1904.29002

**Comment by kotori2** [comment]
@shiena I think it is not a color palette issue. I tried to set color scheme with `.\ColorTool campbell.ini` or `.\ColorTool cmd-legacy.ini` but it still broke.
![image](https://user-images.githubusercontent.com/17663689/62883340-688e9780-bd66-11e9-9d57-fa576fc6fd5b.png)
Also I tried with PS6 and cmd and they all works. But it seems white in PS5 is lighter than that in PS6 and cmd
![image](https://user-images.githubusercontent.com/17663689/62883831-9d4f1e80-bd67-11e9-8bbd-be20096283fb.png)
after setting color scheme, all fonts became yellow in PS5
![image](https://user-images.githubusercontent.com/17663689/62884141-59a8e480-bd68-11e9-9c5c-e85b74f8b220.png)
is that Microsoft's fault?
